src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs

```diff
@@ -515,25 +515,27 @@ private void AddToOpenDocumentMap(DocumentId documentId)
             }
         }
 
-        protected internal void OnAdditionalDocumentOpened(DocumentId documentId, SourceTextContainer textContainer, bool isCurrentContext = true)
+        protected internal void OnAdditionalDocumentOpened(DocumentId documentId, SourceTextContainer textContainer, bool isCurrentContext = true, bool requireDocumentPresentAndClosed = true)
```

Comment:
these are existing protected members.  They cannot be changed (it will break binary compat).